server.js

```diff
@@ -1,35 +1,128 @@
 import express from 'express'
 import bodyParser from 'body-parser'
 import cors from 'cors'
-import mongoose from 'mongoose'
+import mongoose, { MongooseDocument } from 'mongoose'
+import { stringify } from 'querystring'
+import netflixData from './data/netflix-titles.json'
 
 // If you're using one of our datasets, uncomment the appropriate import below
 // to get started!
 // 
 // import goldenGlobesData from './data/golden-globes.json'
 // import avocadoSalesData from './data/avocado-sales.json'
 // import booksData from './data/books.json'
-// import netflixData from './data/netflix-titles.json'
 // import topMusicData from './data/top-music.json'
 
-const mongoUrl = process.env.MONGO_URL || "mongodb://localhost/project-mongo"
+const mongoUrl = process.env.MONGO_URL || "mongodb://localhost/shows"
 mongoose.connect(mongoUrl, { useNewUrlParser: true, useUnifiedTopology: true })
 mongoose.Promise = Promise
 
-// Defines the port the app will run on. Defaults to 8080, but can be 
-// overridden when starting the server. For example:
-//
-//   PORT=9000 npm start
-const port = process.env.PORT || 8080
+
+  const Show = mongoose.model('Show', {
+    showID:{
+      type: Number,
+    },
+    title: {
+      type: String,
+    },
+    directors: {
+      type: String,
+    },
+    cast: {
+      type: String,
+    },
+    country: {
+      type: String,
+    },
+    date_added: {
+      type: String,
+    },
+    release_year: {
+      type: Number,
+    },
+    rating: {
+      type: String,
+    },
+    duration: {
+      type: String,
+    },
+    listed_in: {
+      type: String,
+    },
+    description: {
+      type: String,
+    },
+    type: {
+      type: String,
+    },
+  })
+
+  if (process.env.RESET_DATABASE) {
+    console.log('Resetting database!')
+    
+    const seedDatabase = async () => {
+    
+      await Show.deleteMany()
+
+      await netflixData.forEach((show) => new Show(show).save())
+    
+    }
+  seedDatabase()
+  }
+
+
+const port = process.env.PORT || 5000
 const app = express()
 
 // Add middlewares to enable cors and json body parsing
 app.use(cors())
 app.use(bodyParser.json())
 
 // Start defining your routes here
+
 app.get('/', (req, res) => {
-  res.send('Hello world')
+  res.send('My endpoints= /shows & /shows?query=movie or /shows?query=Tv Show & /shows/:title & /shows/year/:release_year & /shows/country/:country' )
+})
+
+app.get('/shows', async (req, res) => {
+  const { query } = req.query
```

Comment:
I thought this was a general query so I tried to search for chocolate 😎 Can you name it something more descriptive so that the user knows this query is searching for the type of show?